oemoflex/model/datapackage.py

```diff
@@ -239,24 +239,15 @@ def from_metadata(cls, json_file_path):
 
         return cls(dir, data, rel_paths)
 
-    def infer_metadata(self, foreign_keys_update=None):
+    def infer_metadata(self, foreign_keys_update):
```

Comment:
This is not an update of the foreign keys of oemoflex any more, but a replacement of the default foreign keys. I will fix that.

Reply:
Done in [d3de683](https://github.com/rl-institut/oemoflex/pull/46/commits/d3de6833bd6eb1943bc807e86509a2bbcc8aed0c)